### ISSUE TITLE
Profvis profiles do not display

### ISSUE THREAD

**Issue by wch** [rstudio/rstudio#1913]
For example, nothing shows up when I run this on 1.2.226 (Mac):

```R
p <- profvis::profvis({ rnorm(1e6) })
p
```

It does work when I use 1.1.408, though.

My current workaround is to save to a file and open with a browser:

```R
htmlwidgets::saveWidget(p, "profile.html")
```

**Comment by jmcphers** [comment]
Appreciate the report, @wch!

**Comment by dfalty** [comment]
Verified fixed 1.2.359.
